Announce.php

```diff
@@ -103,4 +105,82 @@ function schema() {
 			array( 'UpdateFunction', 'delete_orphans_dismissals' ),
 		);
 	}
+
+	/**
+	 * Add the RESTful routes handled by this plugin.
+	 *
+	 * @param string $p_event_name The event name
+	 * @param array  $p_event_args The event arguments
+	 * @return void
+	 */
+	public function routes( $p_event_name, $p_event_args ) {
+		$t_app = $p_event_args['app'];
+		$t_plugin = $this;
+		$t_app->group(
+			plugin_route_group(),
+			function() use ( $t_app, $t_plugin ) {
+				$t_app->get( '/dismiss/{context_id}', [$t_plugin, 'route_dismiss'] );
```

Comment:
This seems like a `POST` rather than a `GET`.

Reply:
Right.